code/modules/arcade/arcade_prize.dm

```diff
@@ -79,11 +78,12 @@
 
 /obj/item/stack/tickets/update_icon()
 	var/amount = get_amount()
-	if((amount >= 75))
-		icon_state = "tickets_4"
-	else if(amount >=25)
-		icon_state = "tickets_3"
-	else if(amount >= 4)
-		icon_state = "tickets_2"
-	else
-		icon_state = "tickets_1"
+	switch(amount)
```

Comment:
`switch(get_amount())` if you want to be super concise

Reply:
Don't know how I didn't notice